tests/testthat/test-object_usage_linter.R

```diff
@@ -482,13 +482,13 @@ test_that("unknown infix operators give good lint metadata", {
 })
 
 test_that("respects `skip_with` argument for `with()` expressions", {
-  f <- tempfile()
-  writeLines(
-    "test_fun <- function(df) {
-      with(df, first_var + second_var)
-     }",
-    f
-  )
-  expect_true(length(lint(f, object_usage_linter(skip_with = TRUE))) == 0L)
-  expect_true(length(lint(f, object_usage_linter(skip_with = FALSE))) == 2L)
+  f <- withr::local_tempfile(
+    lines =
+      "test_fun <- function(df) {
```

Comment:
better as a character vector of lines IMO

Reply:
Done.